src/psycop_model_training/model_eval/base_artifacts/tables/table_1.py

```diff
@@ -0,0 +1,305 @@
+"""Code for generating a table 1."""
+import warnings
+from pathlib import Path
+from typing import Optional, Union
+
+import pandas as pd
+import wandb
+
+from psycop_model_training.model_eval.base_artifacts.tables.tables import output_table
+from psycop_model_training.model_eval.dataclasses import EvalDataset
+from psycop_model_training.utils.utils import bin_continuous_data
+
+
+class Table1:
+    """Class for generating table 1."""
+
+    def __init__(
+        self,
+        eval_dataset: EvalDataset,
+    ) -> None:
+        self.eval_dataset = eval_dataset.to_df()
```

Comment:
```suggestion
        eval_dataset: EvalDataset,
    ) -> None:
        self.eval_df = eval_dataset.to_df()
```

In general, it's nice to have the argument and the attribute have the same type if they share the same name. Again, _tiny_ changes.

Reply:
That said, we're removing the to_df method, so I'll rewrite this now.